### ISSUE TITLE
Move sender out of signed portion of votes

### ISSUE THREAD

**Issue by ian-algorand** [algorand/go-algorand#137]
Summary: In votes, we should sign the round, period, step, and proposal value, but not the sender. This way all votes in a bundle will include signatures of the same message. This is necessary for signature aggregation in bundles, which is a post-v1 feature, so it doesn't need to happen before launch. This could be delayed until we do signature aggregation. The change is reasonably small.

**Comment by ian-algorand** [comment]
year old task, deferred.
